.github/workflows/ci.yml

```diff
@@ -36,3 +36,34 @@ jobs:
         cache: true
     - name: Run tests
       run: make test
+
+  run-benchdiff:
+    runs-on: ubuntu-latest
+    permissions:
+      checks: write
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
This is required so it can compare agains a particular base-ref, in this particular case `origin/main`